tests/unit/rest/bundles_test.py

```diff
@@ -42,6 +42,7 @@ def test_create(self):
         bundle_id = data[0]["id"]
         data[0]["attributes"].pop("state")
         data[0]["attributes"].pop("state_details")
+        data[0]["attributes"]["metadata"].pop("staged_status", None)
```

Comment:
Is this to fix that test that fails intermittently? Because I've had that issue, too